test/e2e/app-dir/segment-cache/staleness/segment-cache-stale-time.test.ts

```diff
@@ -111,10 +110,10 @@ describe('segment cache (staleness)', () => {
 
     await browser.back()
 
-    // Fast forward 29 seconds. staleTimes.dynamic is configured as 30s, so if
+    // Fast forward 25 seconds. staleTimes.dynamic is configured as 30s, so if
```

Comment:
This was too close to the limit to work reliably when deployed.

Reply:
Nevermind, `fastForward` was not the correct method, because it also affects how timers are fired. Switched to `setFixedTime`, which seems to work as we expect it.